Sources/LiquidLocalDriver/LiquidLocalStorage.swift

```diff
@@ -113,6 +113,18 @@ struct LiquidLocalStorage: FileStorage {
         copy(key: source, to: destination).flatMap { url in delete(key: source).map { url } }
     }
 
+    func getObject(key source: String) -> EventLoopFuture<Data?> {
+        exists(key: source).flatMap { exists in
+            guard exists else {
+                return context.eventLoop.next().makeFailedFuture(LiquidError.keyNotExists)
+            }
+
+            let fileUrl = basePath.appendingPathComponent(source)
+            let data = FileManager.default.contents(atPath: fileUrl.path)
```

Comment:
We should use NIO's file api instead of the FileManager object, since that's non-blocking.